benchmarks/benchmarks/stats.py

```diff
@@ -478,19 +478,20 @@ def time_discrepancy(self, method):
 
 
 class BenchQMCHalton(Benchmark):
-    param_names = ['d', 'scramble', 'n']
+    param_names = ['d', 'scramble', 'n', 'workers']
     params = [
         [1, 10],
         [True, False],
-        [10, 1_000, 100_000]
+        [10, 1_000, 100_000],
+        [1, 4]
     ]
 
     def setup(self, d, scramble, n):
```

Comment:
I guess this is needed to fix it.
```suggestion
    def setup(self, d, scramble, n, workers):
```